tiatoolbox/models/architecture/hovernet.py

```diff
@@ -658,27 +671,32 @@ def postproc(raw_maps: List[np.ndarray]):
         """Post processing script for image tiles.
 
         Args:
-            raw_maps (list(ndarray)): list of prediction output of each head and
-                assumed to be in the order of [np, hv, tp] (match with the output
+            raw_maps (list(:class:`numpy.ndarray`)):
+                A list of prediction output of each head and assumed to
+                be in the order of [np, hv, tp] (match with the output
                 of `infer_batch`).
 
         Returns:
-            inst_map (ndarray): pixel-wise nuclear instance segmentation
-                prediction.
-            inst_dict (dict): a dictionary containing a mapping of each instance
-                within `inst_map` instance information. It has following form
-
-                inst_info = {
-                    box: number[],
-                    centroids: number[],
-                    contour: number[][],
-                    type: number,
-                    prob: number,
-                }
-                inst_dict = {[inst_uid: number] : inst_info}
-
-                and `inst_uid` is an integer corresponds to the instance
-                having the same pixel value within `inst_map`.
+            tuple:
+                - :class:`numpy.ndarray` - Instance map:
+                    Pixel-wise nuclear instance segmentation prediction.
+                - :class:`numpy.ndarray` - Instance dictionary:
+                    A dictionary containing a mapping of each instance
+                    within `inst_map` instance information. It has
+                    following form:
+
+                    >>> inst_info = {
+                    ...     box: number[],
+                    ...     centroids: number[],
+                    ...     contour: number[][],
+                    ...     type: number,
+                    ...     prob: number,
+                    ... }
+                    >>> inst_dict = {[inst_uid: number] : inst_info}
```

Comment:
@vqdang and @adamshephard, `[inst_uid: number]` does not look like valid python to me. What are you trying to say here? That the dictionary key here is a number?